### PR TITLE
Add kwargs to from_loaders

### PR DESCRIPTION
**Pull request by timothyasp**
Most of the from_documents functions have more than a few args. It's really nice using the single line loader, so lets pass along kwargs

### CODE REVIEW

**Review by hwchase17 — COMMENTED** [review]
love this - lets actually put this during init of the class tho? have a vectorstore_kwargs that defaults to empty dict? happy to help as well

**Comment by timothyasp** [comment]
Sounds good to me.  Should probably remove the `embedding` arg as well and just move that into the `vectorstore_kwargs`

**Comment by timothyasp** [comment]
Just took a shot at it @hwchase17  - thoughts?

**Review by hwchase17 — COMMENTED** [review]
i would still keep embeddings as its own thing - its a central enough concept (and also isnt a "kwarg" but an "arg" that i think it should standalone

**Comment by timothyasp** [comment]
Good call - fixed

File: langchain/indexes/vectorstore.py [hunk 1]
```diff
@@ -50,8 +54,9 @@ class VectorstoreIndexCreator(BaseModel):
     """Logic for creating indexes."""
 
     vectorstore_cls: Type[VectorStore] = Chroma
-    embedding: Embeddings = Field(default_factory=OpenAIEmbeddings)
     text_splitter: TextSplitter = Field(default_factory=_get_default_text_splitter)
+    embedding: Embeddings = Field(default_factory=OpenAIEmbeddings)
+    vectorstore_kwargs: dict = Field(default_factory=_get_default_vectorstore_kwargs)
```
Comment (hwchase17) [inline langchain/indexes/vectorstore.py:59]: default_factory should just be `dict` now

File: langchain/indexes/vectorstore.py [hunk 2]
```diff
@@ -18,6 +18,10 @@ def _get_default_text_splitter() -> TextSplitter:
     return RecursiveCharacterTextSplitter(chunk_size=1000, chunk_overlap=0)
 
 
+def _get_default_vectorstore_kwargs() -> dict:
```
Comment (hwchase17) [inline langchain/indexes/vectorstore.py:21]: no longer needed

**Comment by hwchase17** [comment]
ill update this and get it in next release!

**Pull request merged**